### PR TITLE
Updated to .NET 8

### PR DESCRIPTION
**Pull request by daghsentinel**
- Updated target frameworks to .NET 8
- Updated global.json to current SDK
- Updated nuget versions
- Refactored code

### CODE REVIEW

**Comment by daghsentinel** [comment]
I appreciate if you have the time to review this, @bjorkstromm !

File: src/Depends/Depends.csproj [hunk 1]
```diff
@@ -1,24 +1,24 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net6.0;net7.0</TargetFrameworks>
-    <ToolCommandName>dotnet-depends</ToolCommandName>
+    <TargetFrameworks>net8.0</TargetFrameworks>
+    <ToolCommandName>dotnet-dependency</ToolCommandName>
```
Comment (bjorkstromm) [inline src/Depends/Depends.csproj:5]: Please don't change package name

File: src/Depends/Depends.csproj [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
     <PackAsTool>True</PackAsTool>
     <OutputType>Exe</OutputType>
-    <PackageId>dotnet-depends</PackageId>
-    <Title>dotnet-depends</Title>
-    <Authors>mholo65</Authors>
+    <PackageId>dotnet-dependency</PackageId>
```
Comment (bjorkstromm) [inline src/Depends/Depends.csproj:8]: Please don't change package id

File: src/Depends/Depends.csproj [hunk 3]
```diff
@@ -1,24 +1,24 @@
...
+    <Title>dotnet-dependency</Title>
+    <Authors>daghsentinel</Authors>
```
Comment (bjorkstromm) [inline src/Depends/Depends.csproj:10]: please don't change author

File: src/Depends/Depends.csproj [hunk 4]
```diff
@@ -1,24 +1,24 @@
...
-    <ToolCommandName>dotnet-depends</ToolCommandName>
+    <TargetFrameworks>net8.0</TargetFrameworks>
+    <ToolCommandName>dotnet-dependency</ToolCommandName>
     <PackAsTool>True</PackAsTool>
     <OutputType>Exe</OutputType>
-    <PackageId>dotnet-depends</PackageId>
-    <Title>dotnet-depends</Title>
-    <Authors>mholo65</Authors>
+    <PackageId>dotnet-dependency</PackageId>
+    <Title>dotnet-dependency</Title>
```
Comment (bjorkstromm) [inline src/Depends/Depends.csproj:9]: please don't change title

File: src/Depends/Depends.csproj [hunk 5]
```diff
@@ -1,24 +1,24 @@
...
+    <Authors>daghsentinel</Authors>
     <Description>Dependency explorer for .NET</Description>
-    <Copyright>2018 Martin Björkström</Copyright>
+    <Copyright>2024 Dag H. Baardsen</Copyright>
```
Comment (bjorkstromm) [inline src/Depends/Depends.csproj:12]: Please don't change copyright

File: src/Depends/Depends.csproj [hunk 6]
```diff
@@ -1,24 +1,24 @@
...
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
-    <PackageProjectUrl>https://github.com/mholo65/depends</PackageProjectUrl>
+    <PackageProjectUrl>https://github.com/daghsentinel/depends</PackageProjectUrl>
```
Comment (bjorkstromm) [inline src/Depends/Depends.csproj:14]: Please don't change project URL

**Review by bjorkstromm — COMMENTED** [review]
I made some comments. I'll make the changes myself and then merge this PR. Thanks for updating to .NET 8

**Comment by bjorkstromm** [comment]
I'll have a closer look at the refactorings maybe tomorrow.

**Comment by daghsentinel** [comment]
Hi, Thanks! I forgot to correct these. I had the impression that this project was dead, and started out on the journey towards a new fork - forgetting to revert these changes.

Thanks for looking into this!

**Comment by tom-haines** [comment]
Also looking forward to this one :)

**Comment by westdavidr** [comment]
Can we get this one merged? This is a fantastic tool!

**Comment by daghsentinel** [comment]
Looks like a dead project. Again.

**Comment by bjorkstromm** [comment]
Thank you @daghsentinel and sorry for the delay. Life got in the way 😞

**Pull request merged**